Resources/Prototypes/Hydroponics/seeds.yml

```diff
@@ -716,3 +716,50 @@
       Min: 1
       Max: 5
       PotencyDivisor: 20
+
+- type: seed
+  id: rice
+  name: seeds-rice-name
+  noun: seeds-noun-seeds
+  displayName: seeds-rice-display-name
+  plantRsi: Objects/Specific/Hydroponics/rice.rsi
+  productPrototypes:
+    - RiceBushel
+  lifespan: 25
+  maturation: 6
+  production: 1
+  yield: 4
+  potency: 5
+  growthStages: 4
+  idealLight: 5
+  nutrientConsumption: 0.15
+  chemicals:
+    Nutriment:
+      Min: 1
+      Max: 20
+      PotencyDivisor: 20
+    Rice:
+      Min: 5
+      Max: 20
+      PotencyDivisor: 20
+
+- type: seed
+  id: soybeans
+  name: seeds-soybeans-name
+  noun: seeds-noun-seeds
+  displayName: seeds-soybeans-display-name
+  plantRsi: Objects/Specific/Hydroponics/soybeans.rsi
+  productPrototypes:
+    - FoodSoybeans
+  lifespan: 25
+  maturation: 4
+  production: 4
+  yield: 3
+  potency: 5
+  idealLight: 7
+  nutrientConsumption: 0.15
+  chemicals:
+    Nutriment:
+      Min: 1
+      Max: 5
+      PotencyDivisor: 20
```

Comment:
Also, isn't the soybean seed prototype missing the soy milk reagent in the chemicals component? Would it then not produce any like this?

Reply:
Here it is
https://github.com/space-wizards/space-station-14/blob/b474b391c0736f885e3a69fd193de6755478cd98/Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml#L920-L924